desk/app/genuine.hoon

```diff
@@ -0,0 +1,87 @@
+/+  default-agent, verb, dbug, server
+::
+|%
+++  give-payload
+  |=  [id=@ta =simple-payload:http]
+  (give-simple-payload:app:server id simple-payload)
+++  make-secret
+  |=  eny=@uvJ
+  ^-  cord
+  (scot %uw eny)
++$  card  card:agent:gall
++$  versioned-state
+  $%  state-0
+  ==
++$  state-0
+  $:  %0
+      secret=cord
```

Comment:
Arguably we should just store the atom here as-is, and only serialize on an as-needed basis. It'll be smaller to store and easier to reason about. ...Not my very much at all, for either of those, but storing your state in some serialized format feels less clean.